compiler/ast/dag/expr.go

```diff
@@ -62,6 +62,11 @@ type (
 		Kind  string `json:"kind" unpack:""`
 		Value string `json:"value"`
 	}
+	MapCall struct {
+		Kind  string `json:"kind" unpack:""`
+		Expr  Expr   `json:"expr"`
+		Inner Expr   `json:"inner"`
+	}
```

Comment:
Is there a downside to using `Call` rather than adding `MapCall`?

Reply:
Its kind of a different thing than a call. A call is made of args that you get the value of all by passing `this` into it, a map call you evaluate this, then iterate over each value an call the inner eval on each value. So I kind of think this should remain MapCall